app/templates/application.hbs

```diff
@@ -1,12 +1,15 @@
-<div class="pusher">
-  {{nav-bar}}
-  {{#if (eq routing.currentRouteName 'index')}}
-    {{welcome-header}}
-  {{/if}}
-  <div id="main-container" class="ui container main-container {{unless device.isInternetExplorer 'flexed-container' ''}}">
-    {{outlet}}
+{{#side-bar}}
+  <div class="footer-pusher">
```

Comment:
Everything inside `{{#side-bar}} {{/side-bar}}` will go inside the `<div class="pusher"> </div>` defined in `app/templates/components/side-bar.hbs`